pkg/controller/v1beta1/inferenceservice/utils/utils.go

```diff
@@ -288,6 +288,34 @@ func ReplacePlaceholders(container *v1.Container, meta metav1.ObjectMeta) error
 	return json.Unmarshal(buf.Bytes(), container)
 }
 
+// ReplacePlaceholders Replace placeholders in runtime container by values from inferenceservice metadata
+func ReplacePlaceholdersForWorkerSpec(container *v1.Container, workerSpec *v1alpha1.WorkerSpec) error {
+	data, _ := json.Marshal(container)
+	tmpl, err := template.New("container-tmpl").Parse(string(data))
+	if err != nil {
+		return err
+	}
+	buf := &bytes.Buffer{}
+	err = tmpl.Execute(buf, workerSpec)
+	if err != nil {
+		return err
+	}
+
+	err = json.Unmarshal(buf.Bytes(), container)
+	if err != nil {
+		return err
+	}
+
+	if val, ok := container.Resources.Limits[constants.NvidiaGPUResourceType]; ok {
+		qty, err := resource.ParseQuantity(val.String())
+		if err != nil {
+			return err
+		}
+		container.Resources.Limits[constants.NvidiaGPUResourceType] = qty
```

Comment:
what about the other GPUs?

Reply:
this is old code that is not being used by anywhere so I removed.